grafana/common/PG_Overview.json

```diff
@@ -112,14 +112,14 @@
         },
         "textMode": "auto"
       },
-      "pluginVersion": "7.4.5",
+      "pluginVersion": "7.5.12",
       "repeat": "pgnodes",
       "repeatDirection": "h",
       "scopedVars": {
         "pgnodes": {
           "selected": false,
-          "text": "alpha_ip16_pg1",
-          "value": "alpha_ip16_pg1"
+          "text": "golf_ip16_pg1",
```

Comment:
Are these pgnodes names significant changes, or a consequence of testing against a different pg version/cluster that has since serialized out here?

Reply:
No, they are not. They get overwritten with whatever is returned from prometheus after the dashboard is imported.